ci-operator/step-registry/cilium/conf/cilium-conf-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cilium_olm_rev=bdf0e8fd6c5e82708fe9c95adb0d3142e21fabe1
+cilium_version=1.9.5
+
+cat > "${SHARED_DIR}/manifest_cluster-network-03-config.yml" << EOF
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  networkType: Cilium
+  serviceNetwork:
+  - 172.30.0.0/16
+EOF
+
+# Include all Cilium OLM manifest from https://github.com/cilium/cilium-olm/tree/${cilium_olm_rev}/manifests/cilium.v${cilium_version}
+
+curl --silent --location --fail --show-error "https://github.com/cilium/cilium-olm/archive/${cilium_olm_rev}.tar.gz" --output /tmp/cilium-olm.tgz
+tar -C /tmp -xf /tmp/cilium-olm.tgz
+
+cp -a "/tmp/cilium-olm-${cilium_olm_rev}/manifests/cilium.v${cilium_version}/" "${SHARED_DIR}"
```

Comment:
I don't think that worked, at least I don't see any new manifests in `rendered-assets/openshift/manifests`.

Lets add some more debug options (it won't affect the result) - i.e. `ls -la ${SHARED_DIR}` and some more verbosity on the source of the manifests - version, URL, curl output etc.?

Reply:
I have definitely tested this locally quite a few times, will add `ls -la` for now... I am not sure if `${SHARED_DIR}` points at manifest, but we will find out now.